package-managers/qubes-download-dom0-updates.sh

```diff
@@ -72,15 +72,15 @@ fi
 
 mkdir -p $DOM0_UPDATES_DIR/etc
 
-if [ -e /etc/debian_version ]; then
-    # Default rpm configuration on Debian uses ~/.rpmdb for rpm database (as
-    # rpm isn't native package manager there)
-    mkdir -p "$DOM0_UPDATES_DIR$HOME"
-    rm -rf "$DOM0_UPDATES_DIR$HOME/.rpmdb"
-    cp -r "$DOM0_UPDATES_DIR/var/lib/rpm" "$DOM0_UPDATES_DIR$HOME/.rpmdb"
+# Check if we need to copy rpmdb somewhere else
+DBPATH=`rpm -E '%{_dbpath}'`
+if [ ! "$DBPATH" = "/var/lib/rpm" ]; then
+    mkdir -p "$DOM0_UPDATES_DIR$DBPATH"
+    rm -rf "$DOM0_UPDATES_DIR$DBPATH"
+    cp -r "$DOM0_UPDATES_DIR/var/lib/rpm" "$DOM0_UPDATES_DIR$DBPATH"
 fi
 # Rebuild rpm database in case of different rpm version
-rm -f $DOM0_UPDATES_DIR/var/lib/rpm/__*
+rm -f $DOM0_UPDATES_DIR$DBPATH/__*
```

Comment:
```suggestion
rm -f -- "$DOM0_UPDATES_DIR$DBPATH"/__*
```

shellcheck complains

Reply:
why does it want to add -- for one rm, but not the other one? :thinking:

Reply:
That was my suggestion, not ShellCheck’s.  It isn’t strictly necessary.

Reply:
ah! added it to both rm calls while i was at it.